jsr352/core/src/main/java/org/hibernate/search/jsr352/massindexing/MassIndexingJob.java

```diff
@@ -33,7 +33,7 @@
  * 		MassIndexingJob.NAME,
```

Comment:
This Javadoc example is obsolete. Since it's not related to this ticket, I created another ticket for updating it <https://hibernate.atlassian.net/browse/HSEARCH-2824>